src/components/main/Main.js

```diff
@@ -33,6 +33,8 @@ class Main extends Component {
         {
           user
             ? <main className="main">
+              <Tabs days={days} markDone={markDone} delTask={delTask} dayWeek={dayWeek} >
+              </Tabs>
```

Comment:
`Tabs` should get content from its children. There are no props `markDone`, `deltask`, `dayWeek`
```
          <Tabs selectedIndex={...} >
            {
             .days.map((todos, index) => {
                <Tab>
                  <TaskList content={...} markDone={...} delTask={...} />
                </Tab>
              })
            }
          </Tabs>
```